src/protocol.py

```diff
@@ -316,125 +311,6 @@ def assembleErrorMessage(fatal=0, banTime=0, inventoryVector='', errorText=''):
     return CreatePacket('error', payload)
 
 
-# Packet decoding
-
-
-def decryptAndCheckPubkeyPayload(data, address):
```

Comment:
this should be the other way around, remove from shared (after updating class_objectProcessor.py and class_singleWorker.py).

Reply:
OK, I removed it from `protocol` because that copy is broken:
https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/protocol.py#L418

Reply:
I see